Source/BluetoothError.swift

```diff
@@ -28,6 +28,7 @@ public enum BluetoothError: Error {
     case characteristicWriteFailed(Characteristic, Error?)
     case characteristicReadFailed(Characteristic, Error?)
     case characteristicNotifyChangeFailed(Characteristic, Error?)
+    case characteristicStateChangedFailed(Characteristic, Error?)
```

Comment:
I'd prefer adding to this error name an information that it is related to notifications. We will get this error after calling `setNotifyValue` method on `Peripheral` (https://developer.apple.com/documentation/corebluetooth/cbperipheral/1518949-setnotifyvalue), so maybe something like `characteristicSetNotifyFailed` would be better?